formats/json/jvmTest/src/kotlinx/serialization/features/JsonStreamFlowTest.kt

```diff
@@ -29,17 +29,17 @@ class JsonStreamFlowTest {
     val inputString = """{"data":"a"}{"data":"b"}{"data":"c"}"""
     val inputList = listOf(StringData("a"), StringData("b"), StringData("c"))
 
-    @Test
-    fun testEncodeSeveralItems() {
-        val items = inputList
-        val os = ByteArrayOutputStream()
-        runBlocking {
-            val f = flow<StringData> { items.forEach { emit(it) } }
-            f.writeToStream(os)
-        }
-
-        assertEquals(inputString, os.toString(Charsets.UTF_8))
-    }
+//    @Test
+//    fun testEncodeSeveralItems() {
+//        val items = inputList
+//        val os = ByteArrayOutputStream()
+//        runBlocking {
+//            val f = flow<StringData> { items.forEach { emit(it) } }
+//            f.writeToStream(os)
+//        }
+//
+//        assertEquals(inputString, os.toString(Charsets.UTF_8))
```

Comment:
`toString(Charset)` is only supported since Java 10.

We either have to explicitly bump our project's Java requirement for tests or remove this test

Reply:
It's possible to replace it with ` os.toString(Charsets.UTF_8.name())`